Automattic/pocket-casts-android
# Onboarding: Sign in / sign up screen artwork

| 📘 Project: #524  |
| --- |

## Description
This PR displays artwork (currently static images only) on the initial onboarding sign in / sign up screen.

## Testing Instructions

To test this, turn on the onboarding feature flag in `base.gradle`.
- Fresh install and open the app
- Notice that the artwork is displayed on the initial onboarding sign in / sign up screen
- Notice that artwork is adjusted based on whether Google sign in button is shown or not on the screen

## Screenshots or Screencast 

Phone (Pixel 5)
With Google Sign In | Without Google Sign In
----|----
![phone_pixel5_google_sign_in](https://user-images.githubusercontent.com/1405144/204268053-afbeb733-02bf-43c7-ac1f-d62528f35d46.png) | ![phone_pixel5](https://user-images.githubusercontent.com/1405144/204268134-98b6c0e3-6fdf-43b3-a91c-74b319449558.png)


Tablet (Nexus 9)
With Google Sign In | Without Google Sign In
----|----
![tablet_nexus9_with_google_sign_in](https://user-images.githubusercontent.com/1405144/204268180-ff288a68-bbc3-4b37-baff-1444d2d1a6ff.png) | ![tablet_nexus9](https://user-images.githubusercontent.com/1405144/204268255-b89f067e-6edc-48df-95d5-359feff47fa7.png)

## Checklist
- If this is a user-facing change, I have added an entry in CHANGELOG.md
- I have considered whether it makes sense to add tests for my changes
- All strings that need to be localized are in `modules/services/localization/src/main/res/values/strings.xml`
- Any jetpack compose components I added or changed are covered by compose previews
 
#### I have tested any UI changes...

- for accessibility with TalkBack
